support-frontend/assets/components/contributionsReminder/contributionsReminder.jsx

```diff
@@ -31,6 +36,28 @@ const mapStateToProps = state => ({
 
 // ----- Render ----- //
 class ContributionsReminder extends Component<PropTypes, StateTypes> {
+  reminderDates: Array<ReminderDate>;
+  randomIndex: number;
+
+  constructor() {
+    super();
+    this.reminderDates = [
+      {
+        dateName: 'March 2020',
+        timeStamp: '2020-03-19 00:00:00',
+      },
+      {
+        dateName: 'June 2020',
+        timeStamp: '2020-03-01 00:00:00',
+      },
+      {
+        dateName: 'December 2020',
+        timeStamp: '2020-12-01 00:00:00',
+      },
+    ];
+    this.randomIndex = Math.floor(Math.random() * (this.reminderDates.length));
```

Comment:
does this need to happen in a constructor?

Reply:
no, i'll refactor it to be outside the class 👍